torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -15,6 +15,14 @@ inline Tensor elu(Tensor& input, const ELUOptions& options) {
   }
 }
 
+inline Tensor selu(Tensor& input, const SELUOptions& options) {
```

Comment:
I believe `options` can be optional for the functional form, and we should also add a test case for `selu(input)` (i.e. no `options` passed).

Reply:
Just to clarify: should `options` be `c10::optional<SELUOptions>&` or optional by default argument `const SELUOptions& ... = {}` ?

Reply:
My apologies for being unclear - optional by default argument `const SELUOptions& ... = {}` is what I meant.